README.md

```diff
@@ -61,36 +61,36 @@ By default GoReleaser will build the current directory, but you can change the b
 ```yml
 # goreleaser.yml
 # Build customization
-build:
-  binary: drum-roll
-  goos:
-    - windows
-    - darwin
-    - linux
-  goarch:
-    - amd64
+builds:
+  - binary: drum-roll
+    goos:
+      - windows
+      - darwin
+      - linux
+    goarch:
+      - amd64
```

Comment:
Is the legacy `build` syntax also supported? If so maybe it's worth mentioning that it's also still supported.